webrecorder/browsertrix
# Unify crawl and archived item list components

Partially addresses https://github.com/webrecorder/browsertrix-cloud/issues/1483



### Changes

- Use `btrix-archived-item-list` component in Archived Items page
- Update `btrix-crawl-list` to use `btrix-table` with additional columns
- Rename `crawls` -> `archived-items`

### Manual testing

Prerequisite: run `yarn` to install new dependencies
1. Log in as superadmin and run a crawl
2. Go to "Running Crawls". Verify crawls render as expected and are clickable
3. Log out and log back in as regular crawler user
4. Go to "Archived Items". Verify items render as expected
5. Go to "Crawling" -> workflow with crawls -> "Crawls". Verify items render as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| All Running Crawls | <img width="1130" alt="Screenshot 2024-01-22 at 3 37 41 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/e42135c6-6678-40a7-8a24-7bc53b093ee4"> |
| Workflow - Crawls | <img width="1120" alt="Screenshot 2024-01-22 at 4 47 30 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/b9f6e74e-a8b6-48b0-a5a9-6f600e3c6361"> |
| Archived Items | <img width="1126" alt="Screenshot 2024-01-22 at 4 48 22 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/6d7a0493-da76-4e11-a421-a1335688f8ac"> |




### Follow-ups

Consolidate data table styles, tracked here: https://github.com/webrecorder/browsertrix-cloud/issues/1504
